### PR TITLE
chore: migrate check to kokoro and add native image check for monorepo library

### PR DESCRIPTION
**Pull request by mpeddada1**
cc/ @burkedavison

### CODE REVIEW

**Comment by mpeddada1** [comment]
```
Executing: docker run --rm --interactive --network=host --privileged --volume=/var/run/docker.sock:/var/run/docker.sock --workdir=/tmpfs/src --entrypoint=github/java-shared-config/.kokoro/presubmit/downstream-build.sh --env-file=/tmpfs/tmp/tmpvtn5vkna/envfile --volume=/tmpfs:/tmpfs gcr.io/cloud-devrel-kokoro-resources/java8
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "github/java-shared-config/.kokoro/presubmit/downstream-build.sh": permission denied: unknown.
cleanup
```

**Comment by mpeddada1** [comment]
Stacktrace:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project api-common: Fatal error compiling: invalid flag: --release -> [Help 1]
```

File: .kokoro/presubmit/downstream-build.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+cd "${scriptDir}/../.."
+mvn -B -ntp install -Dcheckstyle.skip -Dfmt.skip -DskipTests
+SHARED_CONFIG_VERSION=$(sed -e 's/xmlns=".*"//' pom.xml | xmllint --xpath '/project/version/text()' -)
+
+# Use GCP Maven Mirror
+mkdir -p "${HOME}/.m2"
+cp settings.xml "${HOME}/.m2"
+
+### Round 2
+git clone "https://github.com/googleapis/gapic-generator-java.git" --depth=1
```
Comment (alicejli) [inline .kokoro/presubmit/downstream-build.sh:47]: does this need to be udpated to googleapis/sdk-platform-java?

**Comment by mpeddada1** [comment]
Confirmed that the tests are using `native-maven-plugin:0.9.22` which is defined in the SNAPSHOT of java-shared-config:

```
02:56:39:718 [INFO] Skipping JaCoCo execution due to missing execution data file.
02:56:39:718 [INFO]
02:56:39:718 [INFO] --- native-maven-plugin:0.9.22:test (test-native) @ google-cloud-kmsinventory ---
02:56:39:725 [INFO] ====================
02:56:39:725 [INFO] Initializing project: Google KMS Inventory API
02:56:39:726 [INFO] ====================
02:56:39:727 [INFO] Found GraalVM installation from JAVA_HOME variable
```

File: .kokoro/presubmit/downstream-build.sh (hunk 1 above)
Comment (mpeddada1) [reply .kokoro/presubmit/downstream-build.sh]: Thanks Alice! The original iteration of the script was copied over from sdk-platform-java and was doing some extra work with using the SNAPSHOT version of shared-dependencies which may not be needed for the purpose of testing shared-config. I've cleaned this up a little.

**Comment by mpeddada1** [comment]
For some reason, `graalvm (11, java-orgpolicy) ` is still showing  up despite being removed.

**Review by burkedavison — APPROVED** [review]

**Pull request merged**